test/bigtest/interactors/agreement-form.js

```diff
@@ -1,18 +1,38 @@
 import {
   interactor,
+  isPresent,
+  collection,
   clickable,
   fillable,
   scoped,
   selectable,
+  text,
+  value,
 } from '@bigtest/interactor';
 
 import AgreementFormLinesInteractor from './agreement-form-lines';
 
+@interactor class InternalContactCardInteractor {
+  userName = text('[data-test-user-name]');
+}
+
 export default @interactor class AgreementFormInteractor {
+  isAgreementForm = isPresent('#pane-agreement-form');
   fillName = fillable('#edit-agreement-name');
+  name = value('#edit-agreement-name');
   selectStatus = selectable('#edit-agreement-status');
   fillStartDate = fillable('#period-start-date-0');
   createAgreement = clickable('#clickable-create-agreement');
+  clickCancel = clickable('#clickable-cancel');
+  clickCloseWithoutSave = clickable('[data-test-confirmation-modal-cancel-button]');
+  clickSaveAndClose = clickable('#clickable-update-agreement');
+
+  isInternalContactPresent = isPresent('[data-test-internal-contact]');
```

Comment:
Couldn't a user just `expect(form.internalContacts(0)).is.not.undefined`? Would let us keep internal-contact logic out of the agreement form interactor.